addons/stock/models/stock_picking.py

```diff
@@ -456,7 +462,7 @@ def _compute_json_popover(self):
                 ]
             })
 
-    @api.depends('move_type', 'immediate_transfer', 'move_lines.state', 'move_lines.picking_id')
+    @api.depends('immediate_transfer', 'move_lines.state', 'move_lines.picking_id')
```

Comment:
The 'move_type' of picking is used in method `_get_relevant_state_among_moves` of stock move. maybe you break something with this?

Reply:
Whoops! Adding it back in.